src/redux/Missions/missionsReducer.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const baseUrl = 'https://api.spacexdata.com/v3/missions';
+
+export const getMissions = createAsyncThunk(
+  'getMissions/',
```

Comment:
- [ ] As the previous reviewer mentioned, the app should only load the data from the `rockets` (home page) but as for now the missions also get fetched without navigating to the missions page, kindly consider fixing this it can improve your app performance and get only necessary data

![Screenshot_20221119_084601](https://user-images.githubusercontent.com/25227022/202840772-ccb2ebd6-dd0b-4e99-a477-e565376906ce.png)